src/index.html

```diff
@@ -11,7 +11,188 @@
+        </label>
+
+        <label class="fieldset-option" for="dateOfBirth">
+          Full date of birth:
+          <input
+            type="date"
+            name="date"
+            id="dateOfBirth"
+            required
+          >
+        </label>
+
+        <label for="check">
+          I accept the term of the agreement
+          <input 
+            type="checkbox"
+            name="checkbox"
+            id="check"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label for="email" class="fieldset-option">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label for="password">
+          Password:
+          <input
+            type="password"
+            minlength="5"
+            maxlength="20"
+            name="password"
+            id="password"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="fieldset-option">
+          Do you love cats?
+          <input
+            type="radio"
+            name="choice" 
+            value="yes"
+          >
+          Yes
+          <input
+            type="radio"
+            name="choice"
+            value="no"
+          >
+          No
+        </label>
+        
+        
+        <label for="color" class="fieldset-option">
+          What is your favorite color?
+          <input 
+            type="color"
+            name="color"
+            id="color"
+          >
+        </label>
+
+        <label for="time" class="fieldset-option">
+          What time do you go to bed?
+          <input 
+            type="time"
+            name="time" 
+            id="time"
+          >
+        </label>
+
+        <label for="cars" class="fieldset-option">
+          What are your favorite brand of cars?
+          <select
+            name="cars"
+            id="cars"
+            size="4"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label for="work">
+          How do you rate your work?
+          <input 
+            type="range"
+            name="work"
+            id="work"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label for="comments" class="fieldset-option">
+          Comments:
+          <textarea
+            readonly
+            autocomplete="off"
+            name="comments"
+            id="comments"
+            cols="20"
+            rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
            readonly
            autocomplete="off"
            name="comments"
            id="comments"
            cols="20"
            rows="2"
          ></textarea>
```
![image](https://user-images.githubusercontent.com/97547824/201515827-d1779bdd-f425-4517-8cbc-3b7a3ec4c4e1.png)